src/govuk/components/date-input/date-input.yaml

```diff
@@ -276,26 +276,31 @@ examples:
 - name: classes
```

Comment:
I think the 'no data' example also needs an `id`.

Reply:
Addressed https://github.com/alphagov/govuk-frontend/pull/1990/commits/d0d625ad8eaea1421887eb422f0e64f5b5a56726

Removed the 'no data' example, as adding an id makes it a duplicate of the default example.